recipes/tangled-up-in-unicode/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "tangled-up-in-unicode" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tangled_up_in_unicode-{{ version }}.tar.gz
+  sha256: d28388a93abf448e05c4beb8fe5d4eecfbe7f06f11f6ef15012664f8a67600cb
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+
+test:
+  imports:
+    - tangled_up_in_unicode
+
+about:
+  home: https://github.com/dylan-profiler/tangled-up-in-unicode
+  summary: "Access to the Unicode Character Database (UCD)"
+  description: |
```

Comment:
You need a license type here in order to make sure the package allows redistribution. I don't see license type on pypi nor the repo. Can you please open an issue in the repo so a license file is added?

Reply:
the authors have added a LICENSE file to the development version but the current pypi sources don't have a LICENSE file. Should I hold off this PR until a version with license_file is available on pypi?

Reply:
I've just released a new version on pypi with additional license metadata.

Reply:
@sbrugman I've modified the recipe accordingly for 0.0.4. Would you like to be a maintainer?